docs/geode-native-docs/introduction/developing-solaris.html.md.erb

```diff
@@ -19,25 +19,25 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-This section describes how to build and run a native client application on Solaris.
+This section describes how to build and run a client application on Solaris.
 
 ## <a id="developing-solaris__section_B518757C7D7A405F9E43C1232C2D6B56" class="no-quick-link"></a>Step 1. Set Environment Variables
 
-**Note:** When compiling external projects or applications that are used or referenced by the native client, make sure that you compile them for the same target architecture as your native client installation. For example, if you installed the 32-bit (x86) version of the native client, compile your external projects for 32-bit (x86) architecture.
+**Note:** When compiling external projects or applications that are used or referenced by the client, make sure that you compile them for the same target architecture as your client installation. For example, if you installed the 32-bit (x86) version of the client, compile your external projects for 32-bit (x86) architecture.
 
-Set the native client environment variables on each Solaris host. For each case, *product-dir* is the path to the native client product directory.
+Set the client environment variables on each Solaris host. For each case, *product-dir* is the path to the client product directory.
 
 **For Bourne and Korn shells (sh, ksh, bash)**
 
 <code>
-GFCPP=_product-dir_; export GFCPP<br />
-PATH=$GFCPP/bin:$PATH;export PATH<br />
-LD\_LIBRARY\_PATH=$GFCPP/lib:$LD\_LIBRARY\_PATH;export LD\_LIBRARY\_PATH
+GEODE=_product-dir_; export GEODE<br />
+PATH=$GEODE/bin:$PATH;export PATH<br />
+LD\_LIBRARY\_PATH=$GEODE/lib:$LD\_LIBRARY\_PATH;export LD\_LIBRARY\_PATH
 </code>
 
```

Comment:
The future life of Solaris is very much up in the air, this section could be removed as I don't think it is regarding the "current" version of SunStudio